### ISSUE TITLE
Add "UTC" to time shown in event overview

### ISSUE THREAD

**Issue by tacerus** [openSUSE/calendar-o-o#3]
The times shown in the calendar are always in UTC. To make this more clear, especially when a different timezone was chosen during the event creation/modification process, `UTC` could be appended to the timestamps in the overview page of an event:

![image](https://github.com/openSUSE/calendar-o-o/assets/5121046/59109c3f-3883-4a92-ac9e-1880c08eea8b)

**Comment by hellcp** [comment]
I added a mention in the page footer now

**Comment by tacerus** [comment]
Hi @hellcp, thanks, that is useful, however it does not indicate that the time on /teams/foo/events/bar is UTC. The time zone shown in the footer seems to only be used for the front page?

**Comment by tacerus** [comment]
Sure!

This is my front page:

![image](https://github.com/openSUSE/calendar-o-o/assets/5121046/83dd65e2-6469-4856-bb9c-3f8a831cbf10)

This is one individual event:

![image](https://github.com/openSUSE/calendar-o-o/assets/5121046/a0169bf2-0ac6-4c86-8691-dd15572ddf47)

Both show "All times on this page are in CEST timezone " in the footer, but the presented times are different.

**Comment by tacerus** [comment]
I wonder if it's something special to this event though, because some others (for example "openSUSE Release Engineering Meeting" do show the same time in both places).

**Comment by hellcp** [comment]
It shows the same time in the screenshot, at least for current date, what's on the left is initial date and that's affected by DST. When setting up the event Doug chose the UTC timezone which means that the meeting times are not affected by DST so the two times are divergent (it was CET back in January, it is CEST now).

**Comment by tacerus** [comment]
Apologies if I'm misunderstanding, but if UTC is chosen in the event options, shouldn't the time shown to the user be converted to my time zone (CET or CEST depending on what is currently valid) in both places (the front page and the event page)?
If not, shouldn't the event page indicate that this is UTC and not CEST?

**Comment by hellcp** [comment]
It is translated to the timezone, it's gonna be either CET or CEST depending on the time of year, so a UTC time will be displayed as different values between January and April

**Comment by tacerus** [comment]
Right, but what indicates the time zone? Right now both pages state the times are displayed in my local time zone (CEST), but the time on the event page is not CEST?
